### ISSUE TITLE
Upgrade dependencies

### ISSUE THREAD

**Issue by warpdesign** [warpdesign/react-explorer#241]
Will hopefully fix warning/errors when running `npm install/npm test` in CI/fresh builds.

- [x] webpack v5.x + plugins
- [x] cypress v10.x
- [x] jest v28.x
- [x] npm lockfile

**Comment by AmirHmZz** [comment]
Hey, I'm going to develop a plugin but I cannot build it :( would you please update it and fix building problems?

**Comment by AmirHmZz** [comment]
`npm install` gives me this :
```
 prebuild-install http request GET https://github.com/balena-io-modules/drivelist/releases/download/v9.2.1/drivelist-v9.2.1-electron-v85-win32-x64.tar.gz
    prebuild-install http 404 https://github.com/balena-io-modules/drivelist/releases/download/v9.2.1/drivelist-v9.2.1-electron-v85-win32-x64.tar.gz
    prebuild-install WARN install No prebuilt binaries found (target=11.0.2 runtime=electron arch=x64 libc= platform=win32)
```

**Comment by warpdesign** [comment]
This warning shouldn't prevent the installation: it just means there is no prebuilt binary for your platform. It should build one but you need to have some compiler to be installed for that.

**Comment by warpdesign** [comment]
Fixed in #240
